gen_netdevs.sh

```diff
@@ -1,12 +1,17 @@
 #!/bin/bash
-FILE=gen_netdevs
+WORKDIR=$(pwd)  # Set working directory to the output of pwd
```

Comment:
@Yaxuan-w can you test to see if this works in the current lind container?

Reply:
Yes it works